tests/integration/test_main.py

```diff
@@ -52,20 +63,36 @@ def test_generate_measures(in_memory_sqlite_database, tmp_path):
         environ={},
         user_args=(),
     )
-    assert output_file.read_text() == textwrap.dedent(
-        """\
-        measure,interval_start,interval_end,ratio,numerator,denominator,sex
-        births,2020-01-01,2020-12-31,1.0,1,1,male
-        births,2020-01-01,2020-12-31,0.0,0,1,female
-        births,2021-01-01,2021-12-31,0.0,0,1,male
-        births,2021-01-01,2021-12-31,1.0,1,1,female
-        """
-    )
+    if disclosure_control_enabled:
+        assert output_file.read_text() == textwrap.dedent(
+            """\
+            measure,interval_start,interval_end,ratio,numerator,denominator,sex
+            births,2020-01-01,2020-12-31,,0,0,male
+            births,2020-01-01,2020-12-31,,0,0,female
+            births,2021-01-01,2021-12-31,,0,0,male
+            births,2021-01-01,2021-12-31,,0,0,female
+            """
+        )
```

Comment:
We discussed having richer data here, so that numerators/denominators weren't all replaced with zero.  That's a nice-to-have -- how much work would it be?

Reply:
We did, and I neglected to include them. I have done so now, for this test.